packages/align-deps/src/types.ts

```diff
@@ -27,6 +27,7 @@ export type Options = {
   write: boolean;
   excludePackages?: string[];
   requirements?: string[];
+  diffMode?: string;
```

Comment:
I think we should have the correct type by the time this gets created:
```suggestion
  diffMode?: DiffMode;
```